redirect cron stdout to docker

SamEureka opened the pull request:
@chrisys & @catt-tung ,

Figured I would just clear all the issues while I was in the repo. 🤣 

Just needed to redirect the cron stdout to the docker proc. Also redirected stderr to stdout just in case. 

resolves balenalabs/inkyshot#17

chrisys commented on start.sh:
```diff
@@ -12,7 +12,7 @@ python /usr/app/update-display.py
 
 # Add the job to the crontab using update_hour var, defaulting to 9 AM
-(echo "0 ${UPDATE_HOUR:-9} * * * /usr/app/run-update.sh") | crontab -
+(echo "0 ${UPDATE_HOUR:-9} * * * * /usr/app/run-update.sh > /proc/1/fd/1 2>&1") | crontab -
```
I noticed you added an additional asterisk to the cron expression here - was that intentional?

SamEureka commented:
@chrisys 
Sorry! I found a problem in the crontab entry. I had added an extra * somehow. Fixed and squashed the commits. You'll need to review again.

chrisys commented:
@SamEureka no worries, thanks again for this batch of contributions!

SamEureka commented:
![adventure_time-high_five](https://user-images.githubusercontent.com/4239748/94840377-2e9aad80-03cd-11eb-80b4-341a46b92de7.gif)
